hazelcast-client/src/main/java/com/hazelcast/client/config/ClientProperties.java

```diff
@@ -22,6 +22,16 @@
 public class ClientProperties {
 
     /**
+     * Client shuffles the given member list to prevent all clients to connect to same node when
+     * this property is false. When set to true, client tries to connect to nodes in order as given.
+     */
```

Comment:
Setting it to false means changing the default behaviour. I wanted to leave the behaviour as is to not to surprise users. The users probably relying on client to make balancing in the sense that clients will more or less connect to different nodes, and there will be no bottleneck in one node.

And we added this property because we thought there is added value.(see @enesakar comment) Otherwise we better don't add the property  so that we don't make the properties a mess with unused configurations.